### ISSUE TITLE
Option to trigger property change on JSON editor

### ISSUE THREAD

**Issue by RaphaelAlvez** [ArnoldSmith86/virtualtabletop#1441]
I needed this yesterday for testing some stuff

Not sure if it should be a toggle button or a new type of input. Probably the later.

For example changing the stack offset in holder does not update the holder properly

**Comment by 96LawDawg** [comment]
In your example, do you mean it doesn't change it visibly immediately?  Because if you change it and then drop a new card in the holder, doesn't it work?

**Comment by RaphaelAlvez** [comment]
Yeah but I mean that changes using th JSON editor don't trigger any change routines so testing becomes more difficult.

**Comment by robartsd** [comment]
> Yeah but I mean that changes using th JSON editor don't trigger any change routines so testing becomes more difficult.

I usually just throw in a temporary button and have a routine make the change. Allows for a more repeatable test.

If this were enabled in the JSON editor, you could have it trigger multiple times - the widget gets updated every keystroke when typing as long as the content is still valid JSON.

**Comment by robartsd** [comment]
> For example changing the stack offset in holder does not update the holder properly

I'm not even sure that changing it in a routine will update the child widgets until a new child gets added. This should probably be fixed in Holder.applyDeltaToDOM() as a separate issue.
